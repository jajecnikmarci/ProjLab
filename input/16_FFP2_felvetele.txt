starttest
add room room1 3
add room room2 3 poisonous
add door room1 room2 door1
add item room1 FFP2 ffp2
add person room1 Student student1
printstate room1
printstate student1
pickup student1
printstate student1
printstate room1
gotoroom student1 room2
printstate student1
endtest