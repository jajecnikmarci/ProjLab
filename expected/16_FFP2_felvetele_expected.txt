room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
		ffp2
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student1
	- location: room1
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		poisonImmunity1
	- souls: 3
room1
	- capacity: 3
	- people: 
		student1
	- doors: 
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room1
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		poisonImmunity1
	- souls: 3
